geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/rules/ServerRule.java

```diff
@@ -72,6 +62,41 @@ protected void after() {
     }
   }
 
+  private void startServer() {
+    try {
+      server = null;
+      ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+      String jmxPropertiesFile = classLoader.getResource("test.properties").getPath();
+      int jmxPort = AvailablePort.getRandomAvailablePort(AvailablePort.SOCKET);
+      System.setProperty(PulseConstants.SYSTEM_PROPERTY_PULSE_PORT, Integer.toString(jmxPort));
+      server = Server.createServer(jmxPort, jmxPropertiesFile, jsonAuthFile);
+      server.start();
+    } catch (Exception e) {
+      throw new Error(e);
```

Comment:
Is this "Error" type handled specially somewhere?  Why do we wrap all exceptions in an Error?

Reply:
cleaned out.